coursemapper-kg/recommendation/app/services/course_materials/db/neo4_db.py

```
@@ -871,76 +871,72 @@
+                user_to_concept_relationships = retrieve_user_to_concept_relationships(session, user_id)
+                relationships.extend([r for r in user_to_concept_relationships if r not in relationships])
+
+                for cid in concept_ids:
+                    resource = retrieve_concept_resources(session, material_id, cid)
+                    # logger.info(resource)
+                    relationship = retrieve_resource_to_concept_relationships(session, cid)
+                    # logger.info(relationship)
+                    resources.extend([r for r in resource if r not in resources])
+                    relationships.extend([r for r in relationship if r not in relationships])
+
+            except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)